packages/astro/src/assets/utils/transformToPath.ts

```diff
@@ -1,3 +1,4 @@
+import { deterministicString } from 'deterministic-object-hash';
```

Comment:
I found this package in the backwoods somewhere in a random stack overflow thread, but it's honestly quite nice and does everything I wanted.